api/candlepin-api-spec.yaml

```diff
@@ -9696,3 +9682,33 @@ components:
     basicAuth:
       type: http
       scheme: basic
+
+  parameters:
+    paging_page:
+      name: page
+      description: Number of a page to return
```

Comment:
Not your original wording, but we should use this chance to correct the phrasing here to something like "page [number|index|offset] to return"

Reply:
Done.